src/components/encounter/ohri-encounter-form.tsx

```diff
@@ -284,7 +284,11 @@ export const OHRIEncounterForm: React.FC<OHRIEncounterFormProps> = ({
   useEffect(() => {
     if (invalidFields?.length) {
       setPagesWithErrors(findPagesWithErrors(scrollablePages, invalidFields));
+
       switch (invalidFields[0].questionOptions.rendering) {
+        case 'date':
+          scrollIntoView(invalidFields[0].id, false);
```

Comment:
When the `date` input is invalid and the Datepicker traps focus, it blocks the rest of the UI and won't allow the user to click any other part of the page. Setting the `shouldFocus` boolean property that determines whether to focus on the element or not to `false` assuages this concern.